projects/1.18/assets/epic-fight-mod/epicfight/lang/zh_cn.json

```
@@ -1,174 +1,272 @@
+	"item.epicfight.netherite_longsword": "下界合金长剑",
+	"item.epicfight.stone_spear": "石长矛",
+	"item.epicfight.iron_spear": "铁长矛",
+	"item.epicfight.golden_spear": "金长矛",
+	"item.epicfight.diamond_spear": "钻石长矛",
+	"item.epicfight.netherite_spear": "下界合金长矛",
+	"item.epicfight.stone_greatsword": "石大剑",
+	"item.epicfight.golden_greatsword": "金大剑",
+	"item.epicfight.iron_greatsword": "铁大剑",
+	"item.epicfight.diamond_greatsword": "钻石大剑",
+	"item.epicfight.netherite_greatsword": "下界合金大剑",
+	"item.epicfight.stone_dagger": "石匕首",
+	"item.epicfight.golden_dagger": "金匕首",
+	"item.epicfight.iron_dagger": "铁匕首",
+	"item.epicfight.diamond_dagger": "钻石匕首",
+	"item.epicfight.netherite_dagger": "下界合金匕首",
+	"item.epicfight.skillbook": "技能书",
+	"item.epicfight.glove": "手套",
+
+	"item.minecraft.potion.effect.blooming": "绽放药水",
+	"item.minecraft.splash_potion.effect.blooming": "喷溅型绽放药水",
+	"item.minecraft.lingering_potion.effect.blooming": "滞留型绽放药水",
+
+	"item.epicfight.uchigatana.tooltip": "停止战斗5秒后将会把刀收入刀鞘，并且在下次使用拔刀施展强力攻击",
+
+	"itemGroup.epicfight.items": "史诗战斗物品",
+	
+	"key.epicfight.gui": "史诗战斗-GUI",
+	"key.epicfight.combat": "史诗战斗-战斗",
+	"key.epicfight.show_tooltip": "显示武器进阶技能提示",
+	"key.epicfight.switch_mode": "切换战斗/挖掘模式",
+	"key.epicfight.attack": "攻击",
+	"key.epicfight.weapon_innate_skill": "武器进阶技能",
+	"key.epicfight.skill_gui": "打开技能编辑器",
+	"key.epicfight.dodge": "闪避技能",
+	"key.epicfight.lock_on": "开启锁定",
+	"key.epicfight.mover_skill": "行动技能",
+	"key.epicfight.config": "打开配置界面",
+
+	"skill.epicfight.dancing_edge": "舞刃",
+	"skill.epicfight.dancing_edge.tooltip": "以舞蹈般的剑术砍倒敌人！",
+	"skill.epicfight.battojutsu": "居合",
+	"skill.epicfight.battojutsu.tooltip": "以看不见的速度拔出剑，砍倒敌人。如果你在冲刺时施放，它将以非常高的速度冲刺，碎裂路径上的所有敌人。",
+	"skill.epicfight.steel_whirlwind": "钢铁旋风",
+	"skill.epicfight.steel_whirlwind.tooltip": "向前劈砍并消灭敌人！可蓄力增加攻击范围和速度。",
+	"skill.epicfight.the_guillotine": "刽斧",
+	"skill.epicfight.the_guillotine.tooltip": "用重击处决你的目标。如果目标的生命值低于%s，则立刻处决该目标。",
+	"skill.epicfight.heartpiercer": "穿心",
+	"skill.epicfight.heartpiercer.tooltip": "手持长矛快速猛刺三次。",
+	"skill.epicfight.grasping_spire": "抓握尖塔",
+	"skill.epicfight.grasping_spire.tooltip": "刺入敌人。成功击中敌人后，将进行第二次攻击，根据击中敌人的数量将其击退。",
+	"skill.epicfight.sweeping_edge": "横扫",
+	"skill.epicfight.sweeping_edge.tooltip": "、以强力的斩击劈开附近的敌人。",
+	"skill.epicfight.rushing_tempo": "急速节奏",
+	"skill.epicfight.rushing_tempo.tooltip": "你可以在每次普通攻击结束时额外进行一次攻击。该技能可叠加%s次。",
+	"skill.epicfight.relentless_combo": "无情连击",
+	"skill.epicfight.relentless_combo.tooltip": "释放毁灭性的连击！",
+	"skill.epicfight.liechtenauer": "扎实剑法",
+	"skill.epicfight.liechtenauer.tooltip": "采取%s秒的防御姿势。你可以在攻击时偏移你前方的所有攻击。击杀敌人会增加%s秒的防御姿势持续时间。当此效果生效时，你无法疾跑。",
+	"skill.epicfight.sharp_stab": "尖锐刺击",
+	"skill.epicfight.sharp_stab.tooltip": "无视敌人的防御，以快速而精确地刺击来追击。",
+	"skill.epicfight.eviscerate": "穿心",
+	"skill.epicfight.eviscerate.tooltip": "野蛮地刺击敌人。成功击中敌人后将扯出匕首，造成致命伤害。",
+	"skill.epicfight.blade_rush": "快刀斩",
+	"skill.epicfight.blade_rush.tooltip": "斩杀敌人，施加趔趄状态5秒。趔趄达到3层就会处决目标。",

```

```suggestion
	"skill.epicfight.blade_rush.tooltip": "斜着砍向敌人，施加趔趄状态5秒。趔趄达到3层就会处决目标。",

```
非斩杀，斜着砍向可以修改为别的。对敌人释放斜斩？斩向敌人？